scripts/hid_configurator/NrfHidDevice.py

```diff
@@ -323,6 +324,19 @@ def _discover_module_config(dev, recipient, module_id):
             }
             opt_idx += 1
 
+        if OPT_DESCR_MODULE_TYPE in module_config['options']:
```

Comment:
Please add a comment to explain what this part is about to do

Reply:
Added comment and renamed to module variant (as discussed offline).